sdk/confidentialledger/Azure.Security.ConfidentialLedger/src/ConfidentialLedgerCertificateClientOptions.cs

```diff
@@ -29,7 +31,7 @@ public ConfidentialLedgerCertificateClientOptions(ServiceVersion version = Lates
         {
             Version = version switch
             {
-                ServiceVersion.V2022_05_13 => "2022-05-13",
+                ServiceVersion.V2024_01_26_Preview => "V2024_01_26_Preview",
```

Comment:
How should I interpret this, does this mean all SDK calls will point to the api version V2024_01_26_Preview ?

Reply:
Yes, if the user doesn't set the `ClientCertificateOptions`, the SDK will use the latest version, which is now set to `2024_01_26_preview`.